### ISSUE TITLE
Can we make "public"

### ISSUE THREAD

**Issue by mrustl** [KWB-R/kwb.ucode#1]

**Comment by hsonne** [comment]
No. It contains executable files in `inst/extdata` of which I do not know the origin and whether it is allowed to use them.

**Comment by mrustl** [comment]
Well I was too fast but it should be ok: 

>"This codes documented by the report are public domain, open-source software and can be downloaded from the Internet at URL http://water.usgs.gov/software/ground_water.html/."
>
> [USGS (2008): UCODE_2005 and Six Other Computer Codes for Universal Sensitivity Analysis, Calibration, and Uncertainty Evaluation, page 5](https://igwmc.mines.edu/wp-content/uploads/sites/117/2018/11/UCODE_2005_User_Manual.pdf#page=5)
